examples/grpc-bridge/client/envoy-proxy.yaml

```diff
@@ -22,38 +16,46 @@ static_resources:
           - name: envoy.file_access_log
             typed_config:
               "@type": type.googleapis.com/envoy.config.accesslog.v2.FileAccessLog
-              path: "/var/log/envoy/admin_access.log"
+              path: "/dev/stdout"
           stat_prefix: egress_http
           use_remote_address: true
           route_config:
             name: local_route
             virtual_hosts:
-            - name: grpc
+            - name: backend
               domains:
+              # The gRPC domain is associated with the endpoint from the server?
```

Comment:
probably remove  this comment,  doesnt seem that useful?

Reply:
👍